### ISSUE TITLE
Feature: Node.js Version Declaration

### ISSUE THREAD

**Issue by taylorsteiger-rad** [vercel/terraform-provider-vercel#68]
Creating a Vercel project with Terraform automatically creates it with the most recent version. However, it would be ideal to be able to specify the Node.js version to make sure that it matches local development.

Proposed Definition:
```
resource "vercel_project" "example" {
  name         = "example-project"
  framework    = "nextjs"
  node_version = "16.x"
}
```

**Comment by dglsparsons** [comment]
Hi @taylorsteiger-rad, thanks for raising this. 

I've omitted the ability to set `node_version` as it's possible to configure through the `engines` property in your `package.json` file, which actually takes precedence. It's documented [here](https://vercel.com/docs/runtimes#official-runtimes/node-js/node-js-version). 

Specifically: 
> Defining the node property inside [engines](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) of a package.json file will override the selection made in the Project Settings and print a Build Step warning if the version does not match. Please avoid greater than ranges, such as >14.x or >=14, because this will match semver major versions of Node.js once available which contains breaking changes.

I know this can leave a warning, but I think it's preferable as it leaves just a single location for configuration.

**Comment by taylorsteiger-rad** [comment]
Hey @dglsparsons,

That makes complete sense, and I agree that leaving it to the `package.json` keeps things simple. Thanks for clarifying!

**Comment by dglsparsons** [comment]
This is now released in v2.14 of the TF provider. Please update, and let me know if you encounter any issues. Documentation can be found [here](https://registry.terraform.io/providers/vercel/vercel/latest/docs/resources/project#node_version-1).
